src/utils/db.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('matepostgres', 'postgres', 'lek', {
```

Comment:
Hardcoded database credentials can lead to security issues. Consider using environment variables to store sensitive data.